include/openenclave/internal/utils.h

```diff
@@ -49,6 +49,13 @@ OE_INLINE bool oe_is_ptrsize_multiple(size_t n)
     return (d >= 1 && r == 0);
 }
 
+OE_INLINE bool oe_is_size_multiple_alignment(size_t size, size_t alignment)
```

Comment:
It would be good to rewrite this function so that it does not generate an exception in the first place, instead of relying on the order in which this function must be called.